src/assets/index.js

```diff
@@ -227,7 +227,8 @@ const analyze_click = () => {
             console.log("Successful reception of data!!");
             console.log(data);
         },
-        error: function (data) {
+        error: function (err) {
+	    alert("Something went wrong, Please try again after some time");
             console.log("error");
             console.log(data);
```

Comment:
Now you need to console err instead of data, also correctly indent alert

Reply:
I have changed data to err. The alert msg is already aligned properly

Reply:
check line 233!
Not the alert message, its code (line 231)
(index.js)